### DISCUSSION TITLE
Enable Github Copilot

### DISCUSSION THREAD

**Discussion by nandinimi** [microsoft/SATechnicalOnboarding#79] [Questions]
I'm not able to enable github copilot without buying it. How can I move forward? Can someone help.

**Comment by Sameer1103** [comment]
There is no need to buy a plan for GitHub Copilot. We can access the GitHub student developer pack by validating our student identity card

**Reply by ahsanhabibakik** [reply]
My varsity is not providing institutional email now, however, they used to provide it a couple of years ago.
As a result, I can't access the 'GitHub student developer pack'. What else can I do? I have a student Id card.
